lib/sidebar.jsx

```diff
@@ -27,24 +27,28 @@ var TriangleCorner = React.createClass({
 var Sidebar = React.createClass({
   MENU_ENTRIES: [
     {
-      name: "Teaching Activities",
+      name: "Teaching Materials",
```

Comment:
Hmm I thought @hannahkane and @iamjessklein  said that it was called "Teaching Activites" now? It used to be "Materials" but I changed it in a commit late last week...